lib/ansible/modules/packaging/os/redhat_subscription.py

```diff
@@ -599,6 +611,7 @@ def main():
     consumer_name = module.params["consumer_name"]
     consumer_id = module.params["consumer_id"]
     force_register = module.params["force_register"]
+    refresh = module.params["refresh"]
 
```

Comment:
I think, we can add `autoattach` as module parameter with documentation and examples.